tasks/models.py

```diff
@@ -28,6 +28,29 @@ class Task(models.Model):
     def __str__(self) -> str:
         return self.title
 
+    @classmethod
+    def filterByStatus(cls, status_filter):
```

Comment:
the pythonic way is like:
```suggestion
    def filter_by_status(cls, status_filter):
```